README_SB.md

```diff
@@ -0,0 +1,112 @@
+# Measure Authoring Tool (MAT)
+
+## Installation
+The Measure Authoring Tool (MAT) was designed using many open source products including, the Google Web Toolkit (GWT) 
+framework, Java JDK, MySQL, and Eclipse. GWT allows a developer to write client side code in Java and GWT converts 
+it to JavaScript. The MAT uses MySQL as its backend database server, and the IDE is Eclipse/Intellij for Java EE Developers.
+
+Due to the use of these open source products, a working knowledge of Java development and some research into how 
+the products work with each other in your environment may be necessary. Research from discussions on product forums, 
+help documents, internet searches and knowledge of the local environment where the MAT will be running may all need to 
+be checked if there are errors during the install.
+
+### Configure Java (JDK)
+1.	The application has not been tested with version above Java 1.8; (OPEN JDK is fine)  please ensure this version is 
+in the environment.
+2.	Verify that `JAVA_HOME` and `PATH` system variables are pointing to the proper folder(s).
+3.	For example, in a Windows environment, the `JAVA_HOME` (Environment Variables under Advanced System Settings (
+should point to the Java SDK 1.8.x folder and `PATH` should point to the Java 1.8.x/bin.
+
+### Configure Maven
+1. Install and Configure Maven (3.6.X): https://maven.apache.org/install.html
+
+### Installing Local Maven Dependencies
+Two dependencies that need to be installed locally for the Measure Authoring Tool project to work. You
```

Comment:
We need to update "Two" to "Three" here.

Reply:
Fixed

Reply:
@greenemcg Can we please get this PR in. Chris needs lib directory to run those 3 dependencies for his local setup